src/Query/Conditions/BetweenColumnsConditionBuilder.php

```diff
@@ -64,9 +78,11 @@ protected function escapeColumnName($columnName, array &$params = []): string
      * @param mixed $value
      * @param array $params passed by reference
      *
-     * @return string
+     * @throws Exception|InvalidArgumentException|InvalidConfigException|NotSupportedException
+     *
+     * @return array|string
      */
-    protected function createPlaceholder($value, array &$params): string
+    protected function createPlaceholder($value, array &$params)
```

Comment:
That should always return a string or else code in `build()` will fail.

Reply:
fixed.